webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -2945,6 +2949,14 @@ var userOverride = {
                     $applozic('.km-last-child').append(kmChatInputDiv);
                     $applozic(kmChatInputDiv).append(kmChatInput);
                 }
+                var phoneField = document.getElementById('km-phone');
+                if (phoneField !== null) {
+                    phoneField.addEventListener('keydown', function (e) {
+                        e.target.value = e.target.value.match(
+                            /^([0-9]{0,15})/
+                        )[0];
```

Comment:
instead of an event listener, can we use direct HTML input validation-based approach? the reason is, we are adding this listener but not removing anywhere, so I think, it will stay in Global memory forever?
https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/pattern